src/main/java/com/zepben/evolve/database/sqlite/readers/BaseServiceReader.kt

```diff
@@ -29,7 +29,7 @@ open class BaseServiceReader constructor(protected val getStatement: () -> State
 
     protected inline fun <reified T : SqliteTable> loadEach(
         description: String,
-        crossinline processRow: (T, ResultSet, (String) -> String) -> Boolean
+        processRow: (T, ResultSet, (String) -> String) -> Boolean
```

Comment:
I added crossinline because I thought function arguments were not inlined by default. I misunderstood that it is actually different and the default behaviour is they are inlined, so I told Rob he could remove them.